resources/skins.citizen.preferences/RadioGroup.vue

```
@@ -0,0 +1,165 @@
+				<span class="citizen-pref-card__label">{{ option.label }}</span>
+			</span>
+		</cdx-radio>
+	</div>
+</template>
+
+<script>
+const { defineComponent } = require( 'vue' );
+const { CdxRadio } = mw.loader.require( 'skins.citizen.preferences.codex' );
+
+// @vue/component
+module.exports = exports = defineComponent( {
```

Biome (2.4.4)

[error] A global variable should not be reassigned.

(lint/suspicious/noGlobalAssign)